File: adminMarketingUI/src/app/model/connection-log.ts
export class ConnectionLog {
  private _totalUsersConnected: number;
  private _localDateTime: Date;

  constructor(totalUsersConnected: number, localDateTime: Date) {
    this._totalUsersConnected = totalUsersConnected;
    this._localDateTime = localDateTime;
  }

  get totalUsersConnected(): number {
    return this._totalUsersConnected;
  }

  set totalUsersConnected(value: number) {
    this._totalUsersConnected = value;
  }

  get localDateTime(): Date {
    return this._localDateTime;
  }

  set localDateTime(value: Date) {
    this._localDateTime = value;
  }
}

File: blablamovebackend/src/main/java/fr/polytech/unice/blablamove/teamc/blablamovebackend/model/ConnectionLog.java
package fr.polytech.unice.blablamove.teamc.blablamovebackend.model;

import java.time.LocalDateTime;

/**
 * Class ConnectionLog
 *
 * @author Joël CANCELA VAZ
 */
public class ConnectionLog {
	private LocalDateTime localDateTime;
	private long totalUsersConnected;

	public ConnectionLog(LocalDateTime localDateTime, long totalUsersConnected) {
		this.localDateTime = localDateTime;
		this.totalUsersConnected = totalUsersConnected;
	}

	public LocalDateTime getLocalDateTime() {
		return localDateTime;
	}

	public void setLocalDateTime(LocalDateTime localDateTime) {
		this.localDateTime = localDateTime;
	}

	public long getTotalUsersConnected() {
		return totalUsersConnected;
	}

	public void setTotalUsersConnected(long totalUsersConnected) {
		this.totalUsersConnected = totalUsersConnected;
	}
}
